http.conf

```diff
@@ -20,7 +20,7 @@ server {
       try_files $uri $uri/ /index.html;
     }
 
-    location ^~ /engine/ {
+    location ^~ /lesson/ {
```

Comment:
I think we should make a note, possibly in the PR comments, that when we add another endpoint like `article` we will basically need to copy and paste this entire block but with `location ^~ /article/`

Reply:
Made a `/data/` directory in `toynet-content` to simplify this. I could also parse out `/data/` from the request, but I think it's less error prone to just throw it all in `toynet-content/data` :)